packages/blade/src/components/Typography/BaseText/BaseText.stories.tsx

```diff
@@ -17,10 +19,32 @@ export default {
     name: 'Storybook',
     children: 'Base Text',
   },
+  parameters: {
+    docs: {
+      page: () => (
+        <>
+          <Title />
+          <Subtitle>
+            The BaseText component is a wrapper component based on Blade&apos;s token system and
+            should be used only in rarest exceptions where your Typography needs are not satisfied
+            by Title, Heading or Text typography components(eg: highlighting a word in the Title
+            with gradients in a landing page).
+          </Subtitle>
+          <Title>Example</Title>
+          <Primary />
+          <Title>Properties</Title>
+          <ArgsTable story={PRIMARY_STORY} />
```

Comment:
Same as `ArgsTable` but seems more readable
```suggestion
          <Props story={PRIMARY_STORY} />
```

Reply:
can't find its documentation. can you point me to one?